src/pages/AppConfigure.vue

```diff
@@ -26,6 +26,7 @@ import {
   type ConfigureTab,
   type ConfigureTabType,
 } from '@/utils/constants'
+import { createTransactionSigner } from '@/utils/signerUtils'
```

Comment:
Remove unused import. Why wasn't this caught by our linter?

Reply:
Resolved at https://github.com/arcana-network/developer-dashboard/pull/124/commits/2d52bc688d134b3021527fe121112a32d673d02e